pkg/models/registration.go

```diff
@@ -59,10 +59,8 @@ const (
 // Registration - Defines the registration details
 // on the part of north side export clients
 type Registration struct {
+	BaseObject
```

Comment:
Not sure we should be adding BaseObject back in here. We want to get rid of it, as discussed, and I think a previous PR refactored this type to remove it.

Reply:
Corrected.